docs/howtos/quick-start/setup/_windows-setup.mdx

```diff
@@ -0,0 +1,4 @@
+
+Redis is not officially supported on Windows, so we recommend using Docker.
```

Comment:
Might be prudent to include this video. It's often referenced for this scenario:

https://www.youtube.com/watch?v=_nFwPTHOMIY&ab_channel=Redis

Reply:
Awesome, was not aware of this video. We will get this in there.